float32 radian
---
bool result
string message